### ISSUE TITLE
Information about "Howto fetch available dependency versions for unity manifest"

### ISSUE THREAD

**Issue by JaXt0r** [intelligide/assimp-unity#13]
As mentioned in #11 the version isn't available. No problem. I just recommend adding a hint howto check for available versions.
Will reference PR in a minute.

**Comment by intelligide** [comment]
I rewrote the installation documentation recently, but I'm not going to include a specific version, I don't really want to modify the documentation every time.

**Comment by JaXt0r** [comment]
Agreed. Nevertheless, feel free to merge the hint about version on your upm repository. Took me quite some time to figure it out. I hope it helps others using your solutions too.

I updated my Pull Request.
